src/app/dataset-block/metadata-block/components/event-details/dynamic-events/builders/event-section.builder.ts

```diff
@@ -23,7 +30,7 @@ export abstract class EventSectionBuilder<
     public buildEventRows(
         event: TEvent,
         rowDescriptors: EventRowDescriptorsByField,
-        section: keyof TEvent,
+        section: keyof TEvent | keyof ExecuteQueryEventFragment,
```

Comment:
No no no!!!
Why breaking a beautiful abstraction!

Btw, I've noticed we are using resource schema types as `TEvent`, such as `SetPollingSource`, `SetTransform`, while we should probably be using event fragments intsead, such as `SetPollingSourceEventFragment`, `DatasetTransformFragment`

Reply:
Yes, fixed that. Removed `| keyof ExecuteQueryEventFragment`